src/components/Footer.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { Selected } from '../types/Selected';
+import { ItemsLeft } from './ItemsLeft';
+
+interface Props {
+  allTodos: Todo[];
+  selected: Selected;
+  setSelected: (selected: Selected) => void;
+  clearAll: () => void;
+  completedTodos: number;
+}
+
+const filters = [
+  { title: 'All', type: Selected.All, href: '#/' },
+  { title: 'Active', type: Selected.Active, href: '#/active' },
+  { title: 'Completed', type: Selected.Completed, href: '#/completed' },
+];
```

Comment:
Don't need to create additional array, use `Object.values(Selected)` instead for creating array